packages/ui/src/theme/tokens/components/toggleButton.ts

```diff
@@ -194,6 +195,7 @@ export const togglebutton: ToggleButtonTokens = {
       color: { value: '{colors.overlay.50.value}' },
     },
     _disabled: {
+      backgroundColor: { value: '{colors.transparent.value}' },
```

Comment:
This is to override background from the default. It looks weird for a link to have a background. Also, I was wondering if we need a link variation for ToggleButton. It seems the use case is close to 0.

Reply:
Yeah, I can't think of a what a Link variation of a ToggleButton would be used for.